Bio/SeqIO/FastaIO.py

```diff
@@ -323,7 +387,7 @@ def write_record(self, record):
 
         if self.wrap:
             for i in range(0, len(data), self.wrap):
-                self.handle.write(data[i : i + self.wrap] + "\n")
+                self.handle.write(data[i: i + self.wrap] + "\n")
```

Comment:
I think that change (and the one below) would be counter to black coding style (which TravisCI is probably about to complain about via the flake8 checks). See the CONTRIBUTING.rst file for details.

Reply:
Sorry that was due to VSCode auto-formatting